### ISSUE TITLE
[UNT0019] Redundant indirection codefix is not working for arguments

### ISSUE THREAD

**Issue by sailro** [microsoft/Microsoft.Unity.Analyzers#147]
#### To Reproduce
Unity's FPS Sample:

```csharp
    public void FindReferences()
    {
        m_DiskOrbitParticle = particleInstance.GetComponent<ParticleSystem>();
        DebugUtility.HandleErrorIfNullGetComponent<ParticleSystem, ChargedWeaponEffectsHandler>(m_DiskOrbitParticle, this, particleInstance.gameObject);
    }
```

#### Expected behavior
`particleInstance.gameObject` is detected as a candidate for `UNT0019` (particleInstance is a GO), and a code fix is offered

#### Actual behavior
`particleInstance.gameObject` is detected as a candidate for `UNT0019` (particleInstance is a GO), but *no* code fix is offered
